src/pages/api/event/[id]/repair-request.ts

```diff
@@ -29,7 +29,6 @@ async function getRepairRequests(
     where: { event: { id: id as string } }
   });
```

Comment:
you need to include the images relation here
```javascript
const repairRequests = await prisma.repairRequest.findMany({
    where: {
        event: {
            id: id as string
        }
    },
    include: {
        images: true
    }
});
```

Reply:
Isn't that this is a update query? We will not return the record so maybe we don't need to include image here?

Reply:
This endpoint is to get many repair request for a given event id